src/localization/fi.json

```diff
@@ -143,11 +141,22 @@
   },
 
   "editableFields": {
-    "htmlField": "Html-sisältö",
-    "submitFieldText": "Lähetä-napin teksti",
     "numberField": "Numerokenttä",
     "numberFieldMin": "Aseta minimiarvo",
     "numberFieldMax": "Aseta maksimiarvo"
-  }
+  },
 
+  "addableComponent" : {
+    "label": "label",
+    "html": "html",
+    "text": "text",
+    "radio": "radio",
+    "select": "select",
+    "number": "number",
+    "memo": "memo",
+    "checkList": "checkList",
+    "submit": "submit",
+    "dateTime": "dateTime",
+    "autocomplete": "autocomplete"
```

Comment:
Should these be in finnish?

Reply:
Discussed with Veli-matti before the name of form element could be in English